Improved Polish translation

m-aciek opened the pull request:
Style, punctuation

m-aciek committed:
Improved Polish translation

Style, punctuation

haacked commented:
Thanks! Sorry for the late reply. Do you know other native Polish speakers who can double check your work here? That would be helpful to me as I don't speak Polish.

m-aciek commented:
Sure, I will try to find someone to help and review those changes.

yisonPylkita commented on lang/pl/index.md:
```diff
@@ -1,42 +1,43 @@
-1. PATCH wersji, gdy dokonujesz naprawy błędu, która jest kompatybilna z poprzednimi wersjami.
+1. wersję MAJOR, gdy dokonujesz zmian niekompatybilnych z API,
+1. wersję MINOR, gdy dodajesz nową funkcjonalność, która jest kompatybilna z poprzednimi wersjami,
+1. wersję PATCH, gdy naprawiasz błąd w sposób kompatybilny z poprzednimi wersjami.
```
It should be `gdy naprawiasz błąd nie zrywając kompatybilności z poprzednimi wersjami.`

yisonPylkita commented on lang/pl/index.md:
```diff
@@ -1,42 +1,43 @@
 inkrementują wersję major.
 
-System ten został nazwany przeze mnie "Semantycznym Wersjonowaniem". W systemie tym numery wersji i sposób w jaki zostają zmienione wskazują na tworzący je kod oraz na to, co zostało zmodyfikowane od ostatniej wersji.
+System ten został nazwany przeze mnie „wersjonowaniem semantycznym”. W systemie tym numery wersji i sposób, w jaki zostają zmienione, wskazują na tworzący je kod oraz na to, co zostało zmodyfikowane od ostatniej wersji.
```
These commas are invalid here.

yisonPylkita reviewed (changes requested):
Hi
I am a native polish speaker and colleague of @m-aciek.

Everything beside marked lines seems to be fine.

m-aciek committed:
Deeper approach to style improvement, split into lines (half-way of
document).

m-aciek committed:
Spelling, style.

m-aciek committed:
pl/spec/v2.0.0.md synced with pl/index.md

m-aciek committed:
Reviewed, style improved, lines split of Specification, Why Use & half of FAQ.

m-aciek committed:
Style improved, lines split for FAQ and About.

m-aciek committed:
Minor punctuation.

m-aciek commented:
During last few days I've reworked the translation, making new translations for some parts of the document. I've split the lines to have width of 80, like in original document.

@yisonPylkita, could you review new changes and leave your feedback here again?

yisonPylkita commented:
@m-aciek  Yes, of course. I will check changes tomorrow.

yisonPylkita commented:
There seems to be a lot of changes. I will check them tomorrow.

m-aciek commented:
Sure, thanks!

yisonPylkita reviewed (approved):
I have no objections to this change.  Most of the Readme was re-written and now reading it feels way better.

Great job @m-aciek

haacked commented:
Thanks!
